<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>643</width>
    <height>401</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <widget class="QWidget" name="page">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <widget class="QWidget" name="verticalLayoutWidget_2">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>80</y>
          <width>281</width>
          <height>181</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,1,1,1">
         <item>
          <widget class="QLabel" name="messagelabel">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>显示信息</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>手机号</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="phonelineEdit"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>密码</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="codelineEdit"/>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QPushButton" name="setButton">
           <property name="text">
            <string>登录</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="page_2">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(170, 255, 0);</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="1,3">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QPushButton" name="pushButton">
            <property name="text">
             <string>获取波形数据</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_2">
            <property name="text">
             <string>波形显示</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QStackedWidget" name="stackedWidget_2">
          <widget class="QWidget" name="page_3">
           <property name="styleSheet">
            <string notr="true">background-color: rgb(255, 255, 255);</string>
           </property>
           <widget class="QLineEdit" name="portlineEdit">
            <property name="geometry">
             <rect>
              <x>170</x>
              <y>70</y>
              <width>113</width>
              <height>20</height>
             </rect>
            </property>
            <property name="text">
             <string>8888</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
           <widget class="QPushButton" name="startButton">
            <property name="geometry">
             <rect>
              <x>140</x>
              <y>120</y>
              <width>75</width>
              <height>23</height>
             </rect>
            </property>
            <property name="text">
             <string>启动</string>
            </property>
           </widget>
           <widget class="QPushButton" name="stopButton">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="geometry">
             <rect>
              <x>240</x>
              <y>120</y>
              <width>75</width>
              <height>23</height>
             </rect>
            </property>
            <property name="text">
             <string>停止</string>
            </property>
           </widget>
           <widget class="QLabel" name="label">
            <property name="geometry">
             <rect>
              <x>110</x>
              <y>70</y>
              <width>54</width>
              <height>12</height>
             </rect>
            </property>
            <property name="text">
             <string>端口号</string>
            </property>
           </widget>
           <widget class="QLabel" name="messagelabel_2">
            <property name="geometry">
             <rect>
              <x>190</x>
              <y>40</y>
              <width>54</width>
              <height>12</height>
             </rect>
            </property>
            <property name="text">
             <string>TextLabel</string>
            </property>
           </widget>
          </widget>
          <widget class="QWidget" name="page_4">
           <property name="styleSheet">
            <string notr="true">background-color: rgb(85, 255, 255);</string>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="1,3">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_3" stretch="0,0,0">
              <item>
               <widget class="QPushButton" name="waveButton">
                <property name="text">
                 <string>波形数据获取</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pushButton_5">
                <property name="text">
                 <string>绘制波形</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="comboBox">
                <item>
                 <property name="text">
                  <string/>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>SPO</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>IBP</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>ECG</string>
                 </property>
                </item>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QWidget" name="wavewidget" native="true">
              <property name="styleSheet">
               <string notr="true">background-color: rgb(255, 170, 255);</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
